connectorx/src/transports/postgres_arrowstream.rs

```diff
@@ -40,26 +42,31 @@ macro_rules! impl_postgres_transport {
-                { Time[NaiveTime]            => Time64[NaiveTime]         | conversion auto }
-                { TimestampTz[DateTime<Utc>] => DateTimeTz[DateTime<Utc>] | conversion auto }
-                { UUID[Uuid]                 => LargeUtf8[String]         | conversion option }
-                { Char[&'r str]              => LargeUtf8[String]         | conversion none }
-                { ByteA[Vec<u8>]             => LargeBinary[Vec<u8>]      | conversion auto }
-                { JSON[Value]                => LargeUtf8[String]         | conversion option }
-                { JSONB[Value]               => LargeUtf8[String]         | conversion none }
+                { Float4[f32]                => Float64[f64]                           | conversion auto   }
+                { Float8[f64]                => Float64[f64]                           | conversion auto   }
+                { Numeric[Decimal]           => Decimal[Decimal]                       | conversion auto   }
+                { Int2[i16]                  => Int64[i64]                             | conversion auto   }
+                { Int4[i32]                  => Int64[i64]                             | conversion auto   }
+                { Int8[i64]                  => Int64[i64]                             | conversion auto   }
+                { Bool[bool]                 => Boolean[bool]                          | conversion auto   }
+                { Text[&'r str]              => LargeUtf8[String]                      | conversion owned  }
+                { BpChar[&'r str]            => LargeUtf8[String]                      | conversion none   }
+                { VarChar[&'r str]           => LargeUtf8[String]                      | conversion none   }
+                { Name[&'r str]              => LargeUtf8[String]                      | conversion none   }
+                { Timestamp[NaiveDateTime]   => Date64[NaiveDateTime]                  | conversion auto   }
+                { Date[NaiveDate]            => Date32[NaiveDate]                      | conversion auto   }
+                { Time[NaiveTime]            => Time64[NaiveTime]                      | conversion auto   }
+                { TimestampTz[DateTime<Utc>] => DateTimeTz[DateTime<Utc>]              | conversion auto   }
+                { UUID[Uuid]                 => LargeUtf8[String]                      | conversion option }
+                { Char[&'r str]              => LargeUtf8[String]                      | conversion none   }
+                { ByteA[Vec<u8>]             => LargeBinary[Vec<u8>]                   | conversion auto   }
+                { JSON[Value]                => LargeUtf8[String]                      | conversion option }
+                { JSONB[Value]               => LargeUtf8[String]                      | conversion none   }
+                { Inet[IpInet]               => LargeUtf8[String]                      | conversion none   }
```

Comment:
Same with the comment for `postgres_arrow.rs` above, will `conversion option` work for you case here?